### ISSUE TITLE
Tracking results of the SportsMOT validation set

### ISSUE THREAD

**Issue by Lwzzzzzz** [tstanczyk95/McByte#3]
Hi @tstanczyk95 ,thank you for your outstanding work. 

I am using McByte to obtain the tracking results of the SportsMOT validation set, so I used yolox_x_sports_train.pth.tar instead of yolox_x_sports_mix.pth.tar. But I am not sure if the other pre training weights used by McByte overlap with the SportsMOT validation set. May I ask if there is a data breach issue with my actions?

**Comment by hariouat** [comment]
> Hi [@tstanczyk95](https://github.com/tstanczyk95) ,thank you for your outstanding work.
> 
> I am using McByte to obtain the tracking results of the SportsMOT validation set, so I used yolox_x_sports_train.pth.tar instead of yolox_x_sports_mix.pth.tar. But I am not sure if the other pre training weights used by McByte overlap with the SportsMOT validation set. May I ask if there is a data breach issue with my actions?

Hello, please where we can found this pretrained models?

**Comment by Lwzzzzzz** [comment]
@hariouat  hi, you need to read `McByte/INSTALLATION.md`, which mentions downloading the pre-trained models from their original sources. For example, the Google Drive corresponding to Mixsort: [https://drive.google.com/drive/folders/1pQs1gFC_jG0TlGIUMgf3E0I3OztCvgxI](https://drive.google.com/drive/folders/1pQs1gFC_jG0TlGIUMgf3E0I3OztCvgxI)

**Comment by Lwzzzzzz** [comment]
Hi @tstanczyk95 , just following up on my previous question. I’d really appreciate your advice when you have a moment. Thank you!

**Comment by tstanczyk95** [comment]
Hi @hariouat,

All the instructions, including the pretrained models sources are provided in the installation guidelines [here](https://github.com/tstanczyk95/McByte/blob/main/INSTALLATION.md).

In case of issues or unavailability, be sure to check the last section [here](https://github.com/tstanczyk95/McByte/blob/main/INSTALLATION.md#pretrained-model-backups) for the model backups :)

> > Hi [@tstanczyk95](https://github.com/tstanczyk95) ,thank you for your outstanding work.
> > I am using McByte to obtain the tracking results of the SportsMOT validation set, so I used yolox_x_sports_train.pth.tar instead of yolox_x_sports_mix.pth.tar. But I am not sure if the other pre training weights used by McByte overlap with the SportsMOT validation set. May I ask if there is a data breach issue with my actions?
> 
> Hello, please where we can found this pretrained models?

**Comment by tstanczyk95** [comment]
> Hi [@tstanczyk95](https://github.com/tstanczyk95) ,thank you for your outstanding work.
> 
> I am using McByte to obtain the tracking results of the SportsMOT validation set, so I used yolox_x_sports_train.pth.tar instead of yolox_x_sports_mix.pth.tar. But I am not sure if the other pre training weights used by McByte overlap with the SportsMOT validation set. May I ask if there is a data breach issue with my actions?

Hi @Lwzzzzzz, thank you for your interest and your kind words!

For the most optimal tracker performance as well as the paper result reproduction, please use the models as described in the installation steps [here](https://github.com/tstanczyk95/McByte/blob/main/INSTALLATION.md). 

In case some models are unavailable, check the last section of these guidelines [here](https://github.com/tstanczyk95/McByte/blob/main/INSTALLATION.md#pretrained-model-backups). You can also use this reference in case of ambiguities, which model exactly to use, etc.
